README.md

```diff
@@ -6,15 +6,65 @@ It communicates with the clients, configures the audio devices, and launches the
 
 ## Building
 
-`mrlabctrl` uses `cmake` for building, so compilation works the standard way, e.g.:
+### Prerequisites
+
+_mrlabctrl_ needs a standard C++ toolchain to build and should compile with MSVC, clang, or gcc (on Windows, JUCE is reported to not anymore support the minGW gcc compiler).
+
+On Windows, one way is to install _Visual Studio 2022 (Community)_ with the _Desktop Development with C++_ workload.
+
+### Cloning the repository
+
+Use your IDE's built-in _git_ facility or the command line to clone the repository:
+
+```
+git clone https://github.com/sonible/mrlab-audio-server.git
+```
+
+### Building from the command line
+
+Note that on Windows with Visual Studio, the _Developer Command Prompt_ or the _Developer Power Shell_ should be used in order set the correct environment for the build toolchain.
+
+_mrlabctrl_ uses `cmake` for building and includes _CMake_ preset definitions, so either way of standard cmake commands can be used.
+
+#### Using CMake presets
+
+`cd` into the repository directory.
+
+This will configure and build the main app in _release_ configuration:
+
+```
+cmake --preset release
+cmake --build --preset app-release
+```
+
+There are `debug` and `app-debug` presets for configuration and build in _debug_ configuration, respectively. The `test-release` will build the unit test binary.
+
+The build folders used by the presets are `build-release` and `build-debug` for the respective configurations.
+
+#### Using manual CMake commands
+
+`cd` into the repository directory.
+
+This will configure and build the default targets (i.e., the main app) in _release_ configuration in the `build` subdirectory, using the _Ninja_ generator:
 
 ```
-mkdir build
-cmake -B build -G Ninja -D CMAKE_BUILD_TYPE=Debug
-cd build && ninja
+cmake -B build -G Ninja -D CMAKE_BUILD_TYPE=Release
+cmake --build build
 ```
 
-The project uses [CPM](https://github.com/cpm-cmake/CPM.cmake) for dependency management. It is recommended to specify a cache directory for CPM in order to avoid repeated downloading, e.g.:
+### Building and running in Visual Studio
+
+- Clone the repository from <https://github.com/sonible/mrlab-audio-server.git>.
+- Optionally, select the desired branch to use in the _Git Changes_ tab.
+- If _CMake_ configuration does not take place automatically (might happen after initial checkout), open the project's _Folder View_ in the _Solution Explorer_ to initiate the configuration.
+- After successful _CMake_ configuration, build presets can be selected in the top-central dropdown menu.
+- Build the current target using `Ctrl+B`, start a debug run with `F5` or run without debugging with `Ctrl+F5`.
```

Comment:
So this expects that the project is actually cloned with the built-in git client of vs? I just opened the project folder after cloning it with my standalone git client previously, which also worked well, given that you select "Open a local folder" and not "Open a project or solution" in the startup window. I think this could be described a bit more precisely here and maybe also illustrate the two options which are cloning the repo with vs and opening an already cloned project

Reply:
- [ ] more precise documentation wrt. to cloning.